server.go

```diff
@@ -94,16 +94,28 @@ func New(pathToUnixSocketFile string, configFilePath string) (*KeyManagementServ
 	return keyManagementServiceServer, nil
 }
 
+var kvClient *kv.ManagementClient = nil
```

Comment:
Could you group these variables and move this to the top of the package? The default pointer value will be nil, so we don't need to initialize it again. 

```
var (
  kvClient *kv.ManagementClient
  vaultUrl *string
)
```

Reply:
Done!